python-checks/src/main/java/org/sonar/python/checks/hotspots/ClearTextProtocolsCheck.java

```diff
@@ -47,14 +57,16 @@ public class ClearTextProtocolsCheck extends PythonSubscriptionCheck {
   private static final List<String> SENSITIVE_PROTOCOLS = Arrays.asList("http://", "ftp://", "telnet://");
   private static final Pattern LOOPBACK = Pattern.compile("localhost|127(?:\\.[0-9]+){0,2}\\.[0-9]+$|^(?:0*\\:)*?:?0*1", Pattern.CASE_INSENSITIVE);
   private static final Map<String, String> ALTERNATIVES = new HashMap<>();
+  private static final String SENSITIVE_HTTP_SERVER_CALL = "socketserver.BaseServer.serve_forever";
```

Comment:
Are we sure that in this case only the calls to `serve_forever()` originating from instances of `http.server.HTTPServer`, `http.server.ThreadingHTTPServer` and subclasses will resolve to this fully qualified name? Otherwise it could result in nome FPs.

Reply:
Yes in the worst case scenario it will resolve to the BaseServer which is the parent class of `HTTPServer` and `ThreadingServer`

Reply:
Sounds good.

Reply:
I meant repeating this test, but with an instance of the user-defined `HTTPServer` class. Such as:
```
my_server = MyServer()
my_server.serve_forever()
my_server.server_bind()
```
Maybe it would be an overkill though. Let me know what you think.

Reply:
I see maybe we can discuss what will be tested here if I add the tests.